db/knexfile.js

```diff
@@ -2,11 +2,21 @@
 
 const AppConstants = require("../app-constants");
 
-module.exports = {
+
+// For runtime, use DATABASE_URL
+const RUNTIME_CONFIG = {
   client: "postgresql",
   connection: AppConstants.DATABASE_URL,
-  pool: {
-    min: 2,
-    max: 10,
```

Comment:
Did you want to disable connection pooling for _all_ environments, or just for the "tests" env?

Reply:
These are the `knex` defaults anyway, so I thought I'd just remove them.